### PR TITLE
Add support of apache_request_header

### PR DESCRIPTION
**Pull request by Neofox**
check authorization headers from the method apache_request_header() when
apache won't give the header from the $_SERVER var.

### CODE REVIEW

**Comment by Neofox** [comment]
Pull request for the issue #1

**Review by lvenier — APPROVED** [review]
Good to go ;)

**Comment by rwellens** [comment]
Ha bon le PO/BUM peut valider le code maintenant ? on va garder ce droit seulement pour les anciens CIO.

File: src/ApiGuardian.php [hunk 1]
```diff
@@ -53,11 +53,20 @@ public function __invoke(ApplicationInterface $app)
...
         $request = $app->getRequest();
 
-        if (!$request->hasHeader('Authorization')) {
+        // Fix for issue #1
+        $httpHeader = [];
+        if (\function_exists('apache_request_headers')) {
+            $httpHeader = apache_request_headers();
+        }
+
+        if (!isset($httpHeader['authorization']) && !$request->hasHeader('Authorization')) {
```
Comment (akta54) [inline src/ApiGuardian.php:62]: It should be 'Authorization' instead of 'authorization'.

File: src/ApiGuardian.php [hunk 2]
```diff
@@ -53,11 +53,20 @@ public function __invoke(ApplicationInterface $app)
...
             throw new NoTokenProvidedException();
         }
 
-        if (!\in_array($request->getHeaderLine('Authorization'), $apiKeys, true)) {
+        $authHeader = $request->getHeaderLine('Authorization');
+        $authHeader = empty($authHeader) ? $httpHeader['authorization'] : $authHeader;
```
Comment (akta54) [inline src/ApiGuardian.php:67]: same here

File: src/ApiGuardian.php (hunk 1 above)
Comment (Neofox) [reply src/ApiGuardian.php:62]: no, it's `authorization` `apache_request_headers()` removes all case

**Pull request merged**